doc/release-notes/2023-04-05.rst

```diff
@@ -0,0 +1,88 @@
+.. _release_notes_2023_04_05:
+
+Highlights from 2023-04-05 release
+==================================
+
+Updated client packages are shipped in `Fedora 36`_,  `Fedora 37`_,  `Fedora 38`_
+`EPEL 7`_, `EPEL 8`_, and `EPEL 9`_ Bodhi updates.
+
+
+Disable AppStream for existing Copr projects
+--------------------------------------------
+
+We decided to disable AppStream metadata generation for all project
+excluding those mentioned in `Fedora discussion`_ due to very high I/O
+demand of ``appstream-builder`` tool which causes failures in builds
+even for medium-sized projects. If you want to enable generating AppStream
+data for your project, please go to project settings and enable it back.
+(`#2577`_)
+
+
+Fixing slow searching in Fedora Copr projects
+---------------------------------------------
+
+Searching in Copr was slow and often timeout. We discussed options
+to switch to another search engine since our current search engine
+``whoosh`` seemed to be slow. We managed to speed up the search speed
+by a ton. (`#2534`_)
```

Comment:
By "a ton" sounds a bit overoptimistic, considering we still have to migrate out rather sooner than later.